mackerel-plugin-mysql/lib/mysql.go

```diff
@@ -626,6 +628,25 @@ func (m MySQLPlugin) addExtendedGraphdef(graphdef map[string]mp.Graphs) map[stri
 			{Name: "Handler_savepoint_rollback", Label: "Handler Savepoint Rollback", Diff: true, Stacked: false, Type: "uint64"},
 		},
 	}
+	graphdef["myisam_indexes"] = mp.Graphs{
+		Label: labelPrefix + " myisam Indexes",
```

Comment:
[nit]
Should be `MyISAM`?

Reply:
I think so. But InnoDB related graphs also have `innodb` as label.

Reply:
Oh.
I think there's no reason to use lower-case `innodb` as label...

Reply:
fixed in f4b3759